storybookjs/storybook
# addon-measure: Fix measure not working on disabled elements

[#17762](https://github.com/storybookjs/storybook/issues/17762)

## What I did
Replace `mouseover` and `mousemove` event with `pointerover` and `pointermove` to make measure worked on disabled button

## How to test

- Run yarn storybook:ui in code dir
- Navigate to `http://localhost:6006/?path=/story/storybook-components-button--disabled&globals=measureEnabled:true`
- Verify that `measure` works on disable button
- Demo : https://www.loom.com/share/4be80e174e714cb7adfdb936abd4a8aa
